optimum/exporters/openvino/model_configs.py

```diff
@@ -4537,3 +4537,25 @@ def patch_model_for_export(
         self, model: Union["PreTrainedModel", "TFPreTrainedModel"], model_kwargs: Optional[Dict[str, Any]] = None
     ) -> "ModelPatcher":
         return OVDecoderModelPatcher(self, model, model_kwargs=model_kwargs)
+
+
+@register_in_tasks_manager("dinov3_vit", *["feature-extraction"], library_name="transformers")
+@register_in_tasks_manager("dinov3_convnext", *["feature-extraction"], library_name="transformers")
+class OpenDinoV3OpenVINOConfig(VisionOnnxConfig):
```

Comment:
why not 
```suggestion
class DinoV3OpenVINOConfig(VisionOnnxConfig):
```

Reply:
also could inherit from https://github.com/huggingface/optimum-onnx/blob/fafa2a764401348ee66f17f5b472d58bc67ed520/optimum/exporters/onnx/model_configs.py#L857